setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/CommonFiles.swr

```diff
@@ -657,3 +657,6 @@ folder "InstallDir:MSBuild\Microsoft\VisualStudio\Managed\zh-Hant"
   file source="$(VisualStudioXamlRulesDir)zh-Hant\ResolvedProjectReference.xaml"
   file source="$(VisualStudioXamlRulesDir)zh-Hant\ResolvedSdkReference.xaml"
   file source="$(VisualStudioXamlRulesDir)zh-Hant\SdkReference.xaml"
+
+  folder "Extensions\Microsoft\ManagedProjectSystem\UnifiedSettings"
+    file source="$(VisualStudioExtensionSetupDir)\ProjectSystemSetup\UnifiedSettings\ManagedProjectSystem.registration.json"
```

Comment:
Could we pick this up from output rather than the source project? The file has `CopyToOutputDirectory` so it would be copied somewhere.

Reply:
Or just don't copy it to the output directory.

Reply:
It's a static file, so it's not necessary to pick up from output. @tmeschter removed copying to output dir.